recipes/sfml/all/conanfile.py

```diff
@@ -147,6 +147,9 @@ def winmm():
         def ws2_32():
             return ["ws2_32"] if self.settings.os == "Windows" else []
 
+        def dl():
+            return ["dl"] if self.settings.os == "Linux" else []
```

Comment:
```suggestion
            return ["dl"] if self.settings.os == "Linux" and Version(self.version) >= "2.6.0" else []
```

libdl was not a dependency before 2.6.0 (since https://github.com/SFML/SFML/commit/2eb70c6537d140892d6c05ac7a37688c10fa37cd actually).